rst2pdf/style-to-yaml.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+'''
+    Convert older RSON stylesheets to YAML format
+
+    Run the script with the filename to convert, it outputs to stdout
+'''
+
+import sys
+import json
+import yaml
+from rst2pdf.dumpstyle import fixstyle
+
+from rson import loads as rloads
+
+if __name__ == '__main__':
+    fname = sys.argv[1]
+    # read rson from a file
+    sstr = open(fname, 'rb').read()
```

Comment:
You're leaving this file open. You rarely want to use `open` like this. Use a context manager instead:

```python3
with open(fname, 'rb') as fh:
    style_data = fixstyle(rloads(fh.read())
    # ...
```

Reply:
Ah ha! Thanks